demo/run-osm-demo.sh

```diff
@@ -37,6 +37,7 @@ ENABLE_FLUENTBIT="${ENABLE_FLUENTBIT:-false}"
 DEPLOY_PROMETHEUS="${DEPLOY_PROMETHEUS:-false}"
 DEPLOY_WITH_SAME_SA="${DEPLOY_WITH_SAME_SA:-false}"
 ENVOY_LOG_LEVEL="${ENVOY_LOG_LEVEL:-debug}"
+LOCAL_PROXY_MODE="${LOCAL_PROXY_MODE:-Localhost}"
```

Comment:
Please update `.env.example` on its usage.

Reply:
Example is added: https://github.com/openservicemesh/osm/pull/4752/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR187